torch/onnx/_internal/onnxruntime.py

```diff
@@ -1118,6 +1132,7 @@ def reusable(a: OrtBackendOptions, b: OrtBackendOptions):
                 or a.default_execution_providers != b.default_execution_providers
                 or a.preallocate_output != b.preallocate_output
                 or a.use_aot_autograd != b.use_aot_autograd
+                or a.pre_ort_model_transforms != b.pre_ort_model_transforms
```

Comment:
I'm not sure about the comparison result here, if some transform overwrites the operator ``__eq__``...

Reply:
It might reuse a previously cached session. It can still run.